services/kube-prometheus-stack/44.2.1/defaults/cm.yaml

```diff
@@ -450,15 +468,13 @@ data:
           - name: plugins
             mountPath: /var/lib/grafana/shared-plugins/
     kubeEtcd:
-      # Updated to false due to EKS not allowing the retrieval of data from etcd.
-      enabled: false
-    # kubeEtcd:
-    #   enabled: true
-    #   serviceMonitor:
-    #     scheme: "https"
-    #     caFile: "/etc/prometheus/secrets/etcd-certs/ca.crt"
-    #     certFile: "/etc/prometheus/secrets/etcd-certs/server.crt"
-    #     keyFile: "/etc/prometheus/secrets/etcd-certs/server.key"
+      # enable grafana dashboard for etcd
+      enabled: true
+      # kubeEtcd Service and ServiceMonitor are disabled, etcd metrics are handled by 'etcd-metrics-proxy' component
+      service:
+        enabled: false
```

Comment:
There was [a discussion recently](https://d2iq.slack.com/archives/CR8GEL5TL/p1681286317515369) about this service - does it make sense to have to enable that in `kubeEtcd` given that is used by customers? After giving it some thought, I think we probably shouldn't do that for security reasons.